src/components/widgets/status/StatusLabel.vue

```diff
@@ -35,9 +35,9 @@ export default class AppSwitch extends Mixins(StateMixin) {
     > span {
       display: block;
       // white-space: nowrap;
-      text-overflow: ellipsis;
       overflow: hidden;
-      text-overflow: ellipsis;
+      word-break: break-all;
+      width: auto;
```

Comment:
I don't see this entry actually doing anything, what is the purpose of using `width: auto` here?

Reply:
You are correct. That was left over from my testing. I have removed it.

Reply:
I was referring just to the `width` attribute, but I do agree that `text-overflow` is not working correctly here.